.gitignore

```diff
@@ -27,6 +27,9 @@ idris2docs_venv
 /benchmark/*.csv
 
 /bootstrap-build
+/bootstrap/bin
+/bootstrap/idris2_app/
+/bootstrap/idris2-*/
```

Comment:
These additions seem incorrect to me. There are files in the latter two directories that are intentionally committed to the repository and even if I perform a bootstrap build I don't end up with a bin file/directory or anything other untracked files in the bootstrap directory.

Reply:
I somehow ended up with these untracked files:
```
?? bootstrap/bin/
?? bootstrap/idris2-0.4.0/
?? bootstrap/idris2_app/idris2-boot.so
?? bootstrap/idris2_app/idris2-boot.ss
```
I guess they aren't meant to be there, maybe I messed up a bootstrap run and ended up with some junk. I reverted the change to `.gitignore`.